README.md

```diff
@@ -18,52 +18,43 @@ You can install the ESLint release tool using [npm](https://npmjs.com):
 $ npm install eslint-release --save-dev
 ```
 
-## Local Usage
+## Usage
 
-The ESLint release tool is designed to be used on the command line and has two modes: regular release and prerelease.
+To start, you'll need to define two environment variables:
 
-To run a regular release:
+* `NPM_TOKEN` - a token to use for `npm publish`. The token must be from a user that has permission to publish the package.
+* `ESLINT_GITHUB_TOKEN` - a token for a GitHub user that has `repo` permission (used for posting release notes).
 
-```
-$ eslint-release
-```
+The ESLint release tool is designed to be used on the command line and is divided into two phases: package generation and package publishing.
 
-To run a prerelease, you need to include the prerelease identifier:
+To generate a regular release:
 
 ```
-$ eslint-prerelease alpha
+$ eslint-generate-release
 ```
 
-You can optionally include the release tool in another Node.js script:
+To generate a prerelease, you need to include the prerelease identifier:
 
-```js
-var ReleaseOps = require("eslint-release");
+```
+$ eslint-generate-prerelease alpha
 ```
 
-## CI Usage
-
-When run in a CI environment like Jenkins, the ESLint release tool has different commands that allow for incorporation into more complex release scripts. To start, you'll need to define two environment variables:
-
-* `NPM_TOKEN` - a token to use for `npm publish`. The token must be from a user that has permission to publish the package.
-* `ESLINT_GITHUB_TOKEN` - a token for a GitHub user that has `repo` permission (used for posting release notes).
+Both `eslint-generate-release` and `eslint-generate-prerelease` generate a new version and update the changelog but will not push back to GitHub or publish to npm. It will  generate an npm package and a `.eslint-release-info.json` file.
```

Comment:
Remove extra space.

```suggestion
Both `eslint-generate-release` and `eslint-generate-prerelease` generate a new version and update the changelog but will not push back to GitHub or publish to npm. It will generate an npm package and a `.eslint-release-info.json` file.
```